src/applications/income-and-asset-statement/components/ContactInformationPage.jsx

```diff
@@ -0,0 +1,141 @@
+import { selectProfile } from '~/platform/user/selectors';
+import { VaLink } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+import { formatPhoneNumber } from '../helpers';
+
+const ContactInformation = ({ formData, router, setFormData }) => {
+  const profile = useSelector(selectProfile);
+  const { userFullName, email: profileEmail, phone: profilePhone } =
+    profile || {};
+  const { email, phone } = formData || {};
+
+  // Initialize email and phone with user profile if not already set in formData
+  useEffect(
+    () => {
+      const updatedFormData = { ...formData };
+      let needsUpdate = false;
+
+      if (!email && profileEmail) {
+        updatedFormData.email = profileEmail;
+        needsUpdate = true;
+      }
+
+      if (!phone && profilePhone) {
+        updatedFormData.phone = profilePhone;
+        needsUpdate = true;
+      }
+
+      if (needsUpdate) {
+        setFormData(updatedFormData);
+      }
+    },
+    [email, phone, profileEmail, profilePhone, formData, setFormData],
+  );
+
+  // Since Contact Information always follows Personal Information, it’s a reliable
+  // checkpoint to add veteranFullName from user profile if it exists
+  useEffect(
+    () => {
+      if (
+        formData?.claimantType === 'VETERAN' &&
+        userFullName &&
+        JSON.stringify(formData?.veteranFullName) !==
+          JSON.stringify(userFullName)
+      ) {
+        setFormData({
+          ...formData,
+          veteranFullName: userFullName,
+        });
+      }
+    },
+    [formData, setFormData, userFullName],
+  );
+
+  const editEmailHref = '/contact/information/email';
+  const editPhoneHref = '/contact/information/phone';
+
+  const handleRouteChange = (e, editHref) => {
+    e.preventDefault();
+    router.push(editHref);
+  };
+
+  return (
+    <>
+      <h3 className="vads-u-margin-y--2">
+        Confirm the contact information we have on file for you
+      </h3>
+      <va-card>
+        <h4 className="vads-u-font-size--h3 vads-u-margin-top--0">
+          Current Email
+        </h4>
+        <p className="email">
+          <span
+            className="name dd-privacy-hidden"
+            data-dd-action-name="Veteran's email"
+          >
+            {email}
+          </span>
+        </p>
+        <VaLink
+          href={editEmailHref}
+          label={email ? `Edit email ${email}` : 'Add email address'}
+          text={`${email ? 'Edit' : 'Add'} >`}
+          onClick={e => handleRouteChange(e, editEmailHref)}
+        />
+      </va-card>
+      <va-card class="vads-u-margin-top--3">
+        <h4 className="vads-u-font-size--h3 vads-u-margin-top--0">
+          Current phone number (optional)
+        </h4>
+        <p className="email">
+          <span
+            className="name dd-privacy-hidden"
+            data-dd-action-name="Veteran's phone"
+          >
+            {phone ? formatPhoneNumber(phone) : 'None provided'}
```

Comment:
I'd recommend using [`va-telephone` with a `not-clickable` property set](https://design.va.gov/storybook/?path=/docs/components-va-telephone--docs#not-clickable)